src/abis/index.ts

```diff
@@ -88,4 +91,5 @@ export default <StringMap>{
   governance: GovernanceInterface,
   atoken: StaticATokenInterfcae,
   facadeAct: FacadeActInterface,
+  convexStakingWrapper: ConvexStakingWrapperInterface,
```

Comment:
No need to export it here unless is used on an `addTransaction` call

Reply:
This is needed for the wrapper, otherwise this happens and we get the white screen 
![Uploading image.png…]()